tests/testutils/db.py

```diff
@@ -0,0 +1,62 @@
+"""db.py.
+
+Database level utility class for testing against product_live_time_table.
+"""
+
+from functools import wraps
+import sys
+
+from timed_release import config
+from timed_release.connectors.mysql import base_model
+from timed_release.connectors.mysql import rds_db_engine
+from timed_release.connectors.mysql import rds_db_session
+
+
+def create_product_live_time_table():
+    """Create product_live_time table."""
+    with rds_db_session() as session:
```

Comment:
you dont need to do these things. directly put base_model.metadata.create_all(rds_db_engine)